sklearn/dummy.py

```diff
@@ -51,7 +52,7 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
              The default value of `strategy` has changed to "prior" in version
              0.24.
 
-    random_state : int, RandomState instance or None, optional, default=None
+    random_state : int or RandomState instance, default=None
```

Comment:
The original version is the correct one: `None` has a very specific meaning in this case, and it does not correspond to "Nothing". So it does not have the same meaning as e.g. for the sample_weight parameter where we wouldn't mention `None` as an option

Reply:
That's right. I understood as "`None`" random state. However, it really means that the seed from the clock will be used and, therefore, it must be a valid option.